### DISCUSSION TITLE
how to execute a query

### DISCUSSION THREAD

**Discussion by davfl** [Budibase/budibase#13093] [Help] [answered]
GoodMorning;
I have a query that is called pdf.
I would pass the query result (result in JSON)  in a request of REST API (in body).
Do you know how I can do it? 

<img width="955" alt="es2" src="https://github.com/Budibase/budibase/assets/100681729/98a17ebd-9945-46b3-9eeb-6450adcb3e28">

**Comment by ConorWebb96** [comment, marked as answer]
Going to copy and paste an comment that was sent on this via email. What was suggest is below this may help others who stumble upon this discussion in the future.

You should be able to create a [binding](https://docs.budibase.com/docs/rest-bindings#creating-and-using-bindings), which you can use to pass the returned query into the body. You can use this along with the action [Execute query](https://docs.budibase.com/docs/data-actions#execute-query) to pass the return query result into the rest api. Afterwards you can use the [Update State](https://docs.budibase.com/docs/data-actions#update-state) action to store whatever the post rest request returns and reference this state variable somewhere within the design area to see the result.

**Reply by davfl** [reply]
Thanks for all!!
